src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonObjectCreationHandlingAttribute.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json.Serialization;
+
+/// <summary>
+/// When placed on a field or property, indicates if member will replaced or populated.
```

Comment:
Summary sections typically need to be terse, one-sentence descriptions. You can expand on type vs field/property semantics on the remarks section.

Reply:
New xml comment which will be included in new iteration:

```csharp
/// <summary>
/// Determines how deserialization will handle object creation for properties.
/// </summary>
/// <remarks>
/// When placed on a field or property, indicates if member will replaced or populated
/// and when default resolvers are used this will be mapped to <see cref="JsonPropertyInfo.ObjectCreationHandling"/>.
///
/// When placed on a type with <see cref="JsonObjectCreationHandling.Populate"/> indicates that all members
/// which are capable of being populated will be and when default resolvers are used this will be mapped to <see cref="JsonTypeInfo.PreferredPropertyObjectCreationHandling"/>.
///
/// Note the attribute corresponds only to the preferred values of properties when placed on a type.
///
/// Only property type is taken into consideration. For example if property is of type
/// <see cref="IEnumerable{T}"/> has runtime value of type <see cref="List{T}"/> it will not be populated
/// because <see cref="IEnumerable{T}"/> is not capable of populating.
///
/// Value types require a setter to be populated in which case populating means
/// that deserialization will happen on the copy of the value type and assigned back with setter when deserialization is finished.
/// </remarks>
```